apps/passport-client/.env.example

```diff
@@ -12,3 +12,9 @@ FROGCRYPTO_SERVER_URL=
 # Zupass client will automatically subscribe to all feeds hosted by the Zupass server AND
 # all feeds in the DEFAULT_FEED_URLS array.
 DEFAULT_FEED_URLS='[]'
+
+# License key for the Strich scanner
+STRICH_LICENSE_KEY=""
```

Comment:
what is the pricing of strich? is it per-device or per-scan? if the former it may be worth adding some sort of way to configure which users get strich and which users get the old scanning behavior, perhaps via podbox. if this is something we will need to do, let's create an issue for it and make sure to follow up.

Reply:
I've added details on Notion. Would be interesting to have some statistics on how much scanning is being done and by whom.